challenge3/webapp/toshihiro_yokota/app/controllers/cities_controller.rb

```diff
@@ -0,0 +1,7 @@
+class CitiesController < ApplicationController
+  def index
+    cities = City.eager_load(houses: :energies)
```

Comment:
これ、`houses` を経由せずとも
```suggestion
    cities = City.eager_load(:energies)
```
と書けるのではないか。

Reply:
[13301ba](https://github.com/enechange/EnergyDataSimulationChallenge/pull/283/commits/13301ba8096a8b6e56f27b63934f16bf7851a901) で対応しました。